src/ProcessMaker/Nayra/Bpmn/TokenTrait.php

```diff
@@ -137,4 +139,15 @@ public function getOwnerElement()
     {
         return $this->getOwner()->getOwner();
     }
+
+    /**
+     * Log an error when executing the token
+     *
+     * @param \Throwable $error
+     * @param \ProcessMaker\Nayra\Contracts\Bpmn\FlowElementInterface $bpmnElement
+     */
+    public function logError(Throwable $error, FlowElementInterface $bpmnElement)
```

Comment:
Why this function is throwing an error instead of writing to a log?, Couldn't it be better to log the error and throw it afterwards?

Reply:
For the default case, it is better to throw the exception, this way it could be tested and for the Application (ProcessMaker) this method should be overwritten to save the log in the proper storage